products/git.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Git
+category: app
+iconSlug: git
+permalink: /git
+versionCommand: git version
+changelogTemplate: https://github.com/git/git/blob/master/Documentation/RelNotes/__LATEST__.txt
+releaseDateColumn: true
+activeSupportColumn: false
+eolColumn: Supported
+
+identifiers:
+-   repology: git
+
+auto:
+-   git: https://github.com/git/git.git
+
+releases:
+-   releaseCycle: "2"
+    releaseDate: 2014-05-28
+    eol: false
+    latest: '2.41.0'
+    latestReleaseDate: 2023-06-01
+
+-   releaseCycle: "1"
+    releaseDate: 2005-12-21
+    eol: 2014-12-17
+    latest: '1.9.5'
+    latestReleaseDate: 2014-12-17
+
+---
+
+> [Git](https://git-scm.com/) is a free and open source distributed version
+  control system designed to handle everything from small to very large
+  projects with speed and efficiency.
+
+Only the latest release is actively maintained.
```

Comment:
We could at least say that:


- this is only according to what's written on https://github.com/git/git/blob/master/Documentation/howto/maintain-git.txt
- there is a new release approximately every eight to ten weeks according to https://github.com/git/git/blob/master/Documentation/howto/maintain-git.txt (to be confirmed with the actual cadence of the 2-3 last year)
- git is following semver (not explicitly said on https://github.com/git/git/blob/master/Documentation/howto/maintain-git.txt but they seem to follow the same philosophy).

Reply:
Agree with (1,2), but Git breaks backwards compatibility quite often in minor releases, so we shouldn't say semver. See https://github.com/search?q=repo%3Agit%2Fgit+backward+compatibility++path%3A%2F%5EDocumentation%5C%2FRelNotes%5C%2F%2F&type=code